game/scripts/vscripts/units/swiper.lua

```diff
@@ -89,19 +89,20 @@ function SwiperBossThink()
 	local swipeRange = thisEntity.hFrontswipeAbility:GetCastRange(thisEntity:GetAbsOrigin(), thisEntity)
 
 	if thisEntity.hFrontswipeAbility:IsCooldownReady() then
-		local frontSwipeEnemies = thisEntity.hFrontswipeAbility:FindUnitsInCone(
-			thisEntity:GetAbsOrigin(),
-			thisEntity:GetForwardVector(),
-			swipeRange,
-			swipeRange*2,
+		local frontSwipeEnemies = FindUnitsInRadius(
 			thisEntity:GetTeamNumber(),
+			thisEntity:GetOrigin(), nil,
+			swipeRange,
 			DOTA_UNIT_TARGET_TEAM_ENEMY,
-			DOTA_UNIT_TARGET_ALL,
-			DOTA_UNIT_TARGET_FLAG_NONE,
-			FIND_CLOSEST)
+			DOTA_UNIT_TARGET_HERO,
+			DOTA_UNIT_TARGET_FLAG_MAGIC_IMMUNE_ENEMIES + DOTA_UNIT_TARGET_FLAG_FOW_VISIBLE,
```

Comment:
Why do you find only the visible units? If I'm invisible and in the area I should take damage

Reply:
this particular FindUnits call is used only for deciding if we should cast it. actuall impact searches for all units